syft/frameworks/__init__.py

```diff
@@ -3,19 +3,19 @@
 
 logger = logging.getLogger(__name__)
 
-__all__ = list()
+__all__ = []
 
 if dependency_check.tensorflow_available:
-    from syft.frameworks import tensorflow
+    from syft.frameworks import tensorflow  # noqa: F401
 
     __all__.append("tensorflow")
```

Comment:
I think you can bypass F401 by changing the append lines to use the modules.
```suggestion
    from syft.frameworks import tensorflow

    __all__.append(tensorflow.__name__)
```